src/components/themes/themes.js

```diff
@@ -93,6 +93,15 @@ const themes = {
       color: '#cf1c61',
       borderColor: '#fd8ac0'
     }
+  },
+  pricing: {
+    image: <Image icon="cart" color="green" />,
+    label: 'Download',
```

Comment:
@HeyStenson This will be the default title for the Note so you'll want to switch this label to "Pricing" or something similar.

Reply:
Eep, copy-paste fail. Thanks for catching this!